nameservice/happypath.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+nscli query account $(nscli keys show jack -a) | jq ".value.coins[0]"
```

Comment:
should change to test1 or change the `init.sh` script to reflect these names

Reply:
made those updates in #279

Reply:
ah fire!! haven't pulled master then